### PR TITLE
ansiweather, ant: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/ansiweather.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ansiweather
+
+> Un script Shell pour afficher les conditions météo actuelles dans votre terminal.
+> Plus d'informations : <https://github.com/fcambus/ansiweather>.
+
+- Affiche une prévision avec le système métrique pour les 5 prochains jours dans la ville de Paris, France :
+
+`ansiweather -u {{metric}} -f {{5}} -l {{Paris,FR}}`
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ansiweather.md:8]:
```suggestion
`ansiweather -u {{métrique}} -f {{5}} -l {{Paris,FR}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages.fr/common/ansiweather.md (hunk 1 above)
Comment (Nico385412) [reply pages.fr/common/ansiweather.md:8]: Hi ! 
This time `métrique` is the correct translation for `metric` :p 

But according to the documentation of ansiweather, the `-u` option only take 2 possible values
image

So i think this time we should not translate this placeholder either 😆

Comment (sbrl) [reply pages.fr/common/ansiweather.md:8]: I agree. While `métrique` may be the correct French translation of `metric`, `metric` holds syntactical meaning here so ti cannot be translated without breaking the example.

**Review by sbrl — APPROVED** [review]
Thanks for the translations, @Nico385412!

File: pages.fr/common/ansiweather.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Affiche une prévision avec des symboles et les données d'ensoleillement sur votre position actuelle :
+
+`ansiweather -s {{true}} -d {{true}}`
+
+- Affiche une prévision avec les données sur le vent et l'humidité sur votre position actuelle :
```
Comment (nicokosi) [inline pages.fr/common/ansiweather.md:14]: I suggest preventing using several times `sur` in the same sentence :

```suggestion
- Affiche une prévision avec des symboles et les données d'ensoleillement pour votre position actuelle :

`ansiweather -s {{true}} -d {{true}}`

- Affiche une prévision avec les données sur le vent et l'humidité pour votre position actuelle :
```

**Review by nicokosi — APPROVED** [review]
Seems good to me. 👍
PS: see my minor optional suggestion. 😊

File: pages.fr/common/ansiweather.md (hunk 2 above)
Comment (Nico385412) [reply pages.fr/common/ansiweather.md:14]: I'm not sure about `pour`. Most of the time I use/hear `sur` about the localisation. 

Like in the army when they say `Restez sur vos positions`

But `pour` works in that case, i'll change it :D

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Pull request merged**